test/IDE/complete_operators.swift

```diff
@@ -259,9 +260,20 @@ func testInfix15<T: P where T.T == S2>() {
 func testInfix16<T: P where T.T == S2>() {
   T.foo#^INFIX_16^#
 }
+
 func testInfix17(x: Void) {
   x#^INFIX_17^#
 }
+
+// VOID_OPERATORS: Begin completions
```

Comment:
Yup.  Took me a bit to figure out, but yeah SourceKit code completion was offering suggestions of all the equality/comparison ops that were added, but it wasn't expecting any suggestions so this test was failing.